db/seeds.rb

```diff
@@ -5,3 +5,28 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+#
+require 'faker'
+
+password = 'password'
+
+user = User.create(
+  username: Faker::Internet.user_name,
+  password: password
+)
+
+billing_infos = BillingInfo.create(
+  last4: Faker::Number.number(4),
+  country: Faker::Address.country,
+  brand: Faker::Business.credit_card_type,
+  user_id: User.last.id { |users| (p users.id)}
+)
+
+books = Book.create(
```

Comment:
@SemyzSerhiy it will be more human-readable to use `book` instead of `books` name for this variable.